0-ad-history-mod-package/simulation/templates/mixins/encyclopedia/encyclopedia_siege_oxybeles.xml

```diff
@@ -4,7 +4,7 @@
     <History>
         The oxybeles was a type of ancient Greek bolt-shooting device developed sometime around the 4th century BC. A predecessor of the ballista, the oxybeles was basically an oversized crossbow (gastraphetes) mounted on a tripod. As the name (translating to “sharp arrow”) suggests, it fired big arrows or javelins. The oxybeles' immense draw weight was pulled with the help of a winch mechanism. While this slowed down reloading times, it also equipped the weapon with enormous power. Its range exceeded several hundred feet (hundreds of meters) and was capable of piercing any soldier's shield, armor, and, at close range, even his body. Without a doubt, the oxybeles was the most dangerous anti-personnel weapon of its time.
  
-However, there were trade-offs for its power. Besides the long reloading time, it was also very heavy, inflexible, and cumbersome. Transportation required a lot of effort.
+However, there were trade-offs for its power. Besides the long reloading time, it was also very heavy, inflexible, and cumbersome. Transportation requires a lot of effort.
```

Comment:
I think it should be 'required'. We've always used the past tense for things of the past - even if the statement would still be true today.